src/domTree.js

```diff
@@ -344,8 +336,164 @@ class symbolNode {
     }
 }
 
+/**
+ * SVG nodes are used to render stretchy wide elements.
+ */
+class svgNode {
+    constructor(children, width, height, x, viewBoxWidth, viewBoxHeight, align) {
```

Comment:
I think combining all of the arguments after `children` into a single arg called `attributes` would simplify things greatly.  This should simplify `toNode` and `toMarkup` b/c we can iterate through `attributes` instead requiring all the `if` statements.  One slight downside is that now the caller will have to provide `viewBox` as a single value, but I think that's a small price to pay.

Reply:
Okay, I'll do that.